include/tvm/ir/transform.h

```diff
@@ -161,7 +161,7 @@ class PassContextNode : public Object {
  */
 class PassContext : public ObjectRef {
  public:
-  PassContext() {}
+  PassContext() : PassContext(make_object<PassContextNode>()) {}
```

Comment:
use https://github.com/apache/incubator-tvm/blob/master/include/tvm/runtime/object.h#L712 and add  `PassContextNode* operator->()`; you will need copy constructor and other things now that default constructor is defined in a different way.